src/commands/txCommands/status.js

```diff
@@ -0,0 +1,21 @@
+const Rx = require('rxjs');
+const txlib = require('./util');
+const chalk = require('chalk');
+
+module.exports = {
+    command: 'status',
+    description: 'get translation status of resources (branches)',
+    handler: argv => {
+        txlib.checkEnvVars();
+        console.log(chalk.blue('checking resource status\n'));
+
+        console.log('Incomplete Resources')
+        txlib.resourcesIncomplete$
+            .do(resource => { console.log(resource[0], resource[1]) })
```

Comment:
if you destructure the `resource` arg, you can name the parts to make it clear what is being logged:

```js
.do(([branchName, percentage]) => console.log(branchName, percentage))
```